src/features/singleRight/delegate/ChooseServicePage/ChooseServicePage.tsx

```diff
@@ -192,15 +207,29 @@ export const ChooseServicePage = () => {
         onRemoveClick={() => {
           onRemove(resource.identifier);
         }}
-        notDelegableCode={status === 'NotDelegable' ? details : undefined}
+        // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+        errorText={t(`single_rights.${localizedErrorCode}_title`)}
+        compact={isSm}
       >
-        <p>{resource.description}</p>
-        <p>{resource.rightDescription}</p>
+        {localizedErrorCode && (
+          <Alert
+            severity='danger'
+            elevated={false}
+            className={classes.notDelegableAlert}
+          >
+            <Heading size='xsmall'>{t(`single_rights.${localizedErrorCode}_title`)}</Heading>
+            <Paragraph>{t(`single_rights.${localizedErrorCode}`)}</Paragraph>
+          </Alert>
+        )}
+        <div>
```

Comment:
```suggestion
        <div className={classes.serviceResourceContent}>
```
I think you forgot to add back the classname for this div, which handles the styling for the gap.

Also, why not add the Alert into this div?

Reply:
Think it looks a bit weirder but I'll add Alert into the div then

Reply:
You can ofc change the styling of the class

Reply:
Or the notDelegableAlert